build/debian-base/Dockerfile

```diff
@@ -12,8 +12,96 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM scratch
+ARG BUILD_BASE
```

Comment:
@justaugustus I know you requested to do a `git rm Dockerfile{.build,}` or such, to see a nicer diff. However, git / GitHub show the diff always the same if I change/delete the two files in one commit.